### DISCUSSION TITLE
Make it possible to adjust quote types

### DISCUSSION THREAD

**Discussion by trondhindenes** [outline/outline#8158] [Ideas]
I just discovered that Outline uses "smart quotes" which look different for the begginning and end quote:

To my eyes this looks horrendous and reminds me of MS word which tries to do all kinds of things to my quotes, none if which are things I want. It would be fantastic if Outline had an option either on a per-user basis or per "workspace" to use regular quotes.
![Screenshot from 2024-12-28 10-03-33](https://github.com/user-attachments/assets/cd5cf6c7-de99-42ae-a86f-442145b9c849)

**Comment by tommoor** [comment]
Disable this setting:

**Settings -> Preferences -> Behavior -> Smart text replacement**
